src/shared/components/mutationTable/column/DbsnpColumnFormatter.tsx

```diff
@@ -1,169 +1,97 @@
 import * as React from 'react';
-import { Circle } from 'better-react-spinkit';
 import 'rc-tooltip/assets/bootstrap_white.css';
+
 import { Mutation } from 'cbioportal-ts-api-client';
 import {
-    DefaultTooltip,
-    TableCellStatusIndicator,
-    TableCellStatus,
-} from 'cbioportal-frontend-commons';
-import { MyVariantInfoAnnotation } from 'genome-nexus-ts-api-client';
-import GenomeNexusMyVariantInfoCache, {
-    GenomeNexusCacheDataType,
-} from 'shared/cache/GenomeNexusMyVariantInfoCache';
+    getMyVariantInfoAnnotation,
+    IMyVariantInfoIndex,
+} from 'cbioportal-utils';
+import { MyVariantInfo, VariantAnnotation } from 'genome-nexus-ts-api-client';
+import {
+    Dbsnp,
+    dbsnpDownload,
+    dbsnpSortValue,
+    RemoteData,
+} from 'react-mutation-mapper';
 
 export default class DbsnpColumnFormatter {
     public static renderFunction(
         data: Mutation[],
-        genomeNexusCache: GenomeNexusMyVariantInfoCache | undefined
+        indexedVariantAnnotations?: RemoteData<
+            { [genomicLocation: string]: VariantAnnotation } | undefined
+        >,
+        indexedMyVariantInfoAnnotations?: RemoteData<
+            IMyVariantInfoIndex | undefined
+        >
     ) {
-        const genomeNexusCacheData = DbsnpColumnFormatter.getGenomeNexusDataFromCache(
-            data,
-            genomeNexusCache
-        );
         return (
             <div data-test="dbsnp-data">
-                {DbsnpColumnFormatter.getDbsnpViz(genomeNexusCacheData)}
+                <Dbsnp
+                    className=""
+                    mutation={data[0]}
+                    indexedVariantAnnotations={indexedVariantAnnotations}
+                    indexedMyVariantInfoAnnotations={
+                        indexedMyVariantInfoAnnotations
+                    }
+                />
             </div>
         );
     }
 
-    private static getGenomeNexusDataFromCache(
```

Comment:
most of the code here refactored into `packages/react-mutation-mapper/src/component/column/Dbsnp.tsx`